var/spack/repos/builtin/packages/aluminum/package.py

```diff
@@ -45,4 +47,14 @@ def cmake_args(self):
             '-DALUMINUM_ENABLE_CUDA:BOOL=%s' % ('+gpu' in spec),
             '-DALUMINUM_ENABLE_MPI_CUDA:BOOL=%s' % ('+mpi_cuda' in spec),
             '-DALUMINUM_ENABLE_NCCL:BOOL=%s' % ('+nccl' in spec)]
+
+        # Add support for OS X to find OpenMP
+        if (self.spec.satisfies('%clang')):
+            if (sys.platform == 'darwin'):
```

Comment:
You can simplify this into a single if statement using `spec.satisfies('%clang platform=darwin')`